src/passwordcheck.js

```diff
@@ -28,7 +28,7 @@ define(['bloomfilter',
         PASSWORD_TOO_SHORT: 'PASSWORD_TOO_SHORT'
       };
 
-      function strengthCheck(password) {
+      function isStrongEnough(password) {
         // Check for passwords that at least contain a number and an alphabet,
         // or if alphabets, then at least (minLength + 4) characters long
         var regexString = '((?=.*[a-zA-Z])(?=.*[0-9])[A-Za-z0-9!@#$%^&*()_+ ]{' + minLength + ',' + (minLength + 4) + '})|([A-Za-z0-9!@#$%^&*()_+ ]{' + (minLength + 4) + ',})';
```

Comment:
@TDA - I think we should change this regexp to check if the string is *only* characters and numbers. If so, check that it is fewer characters than minLength + 4. That'll significantly simplify the regexp, and it'll allow international users to use their special characters as part of the password that do not fit into this list.